Annex60/ThermalZones/ReducedOrder/Validation/VDI6007/package.order

```diff
@@ -1,7 +1,4 @@
 TestCase1
-TestCase10
```

Comment:
Let's do the model rename in a separate branch and merge it after the release. I will merge this issues because of the change in `ATot`.